src/pcapi/routes/adage_iframe/bookings.py

```diff
@@ -50,3 +53,10 @@ def book_educational_offer(authenticated_email: str, body: BookEducationalOfferR
         raise ApiErrors({"educationalYear": "Aucune année scolaire ne correspond à cet évènement"})
 
     return BookEducationalOfferResponse(bookingId=booking.id)
+
+
+@blueprint.adage_iframe.route("/authenticate", methods=["GET"])
+@spectree_serialize(api=blueprint.api, on_success_status=204, on_error_statuses=[401])
+@adage_jwt_required
+def authenticate(authenticated_information: AuthenticatedInformation) -> str:
+    return ""
```

Comment:
On en fait rien pour l'instant du coup ?

Reply:
Pour une 204 tu peux mettre enlever la str retournée ;)
```suggestion
def authenticate(authenticated_information: AuthenticatedInformation) -> None:
    pass
```